contracts/IndexedOrderedSetLib.sol

```diff
@@ -11,7 +11,7 @@ library IndexedOrderedSetLib {
   }
 
   modifier requireValue(IndexedOrderedSet storage self, bytes32 value) {
-    require(contains(self, value));
+    require(contains(self, value), "ethpm@setLib:set-does-not-contain-val");
```

Comment:
`escape:IndexedOrderedSetLib:set-missing-value`

Reply:
`value-not-found`?

Reply:
`value-not-found-in-set`?

Reply:
Worried that `set-missing-value` sounds imperative  rather than descriptive, although it might be correct none the less.